gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -49,7 +49,7 @@
     </module>
     <module name="LineLength"> <!-- Java Style Guide: No line-wrapping -->
         <property name="max" value="120"/>
-        <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
+        <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://|\* \{@link"/>
```

Comment:
I think just the `{@link` piece is sufficient, no need to match `* {@link`, this way we match `a href` handling.

Reply:
imo we should still encourage shorter lines. i.e., something like 
```
/* My amazing function runs the eternal engine. See {@link com.superlongcorpname.thebestteamever.AmazingEternalEngine#runThisSuperDuperAmazingEngine}
```

should still be forced to look like

```
/* My amazing function runs the eternal engine. See 
/* {@link com.superlongcorpname.thebestteamever.AmazingEternalEngine#runThisSuperDuperAmazingEngine}
```

Reply:
I get that, however consistency is more important with this sort of thing than optimality.

For example, how would this work in the context of a list in javadoc (or other nested structure used in high-quality documentation).

If we want to bike-shed on optimality, the following results in the smallest maximum line length, but isn't covered by either suppression (which is fine imo because it matches the behavior of `a href`!)

```java
/**
 * My amazing function runs the eternal engine. See {@link
 * com.superlongcorpname.thebestteamever.AmazingEternalEngine#runThisSuperDuperAmazingEngine
 * }
 */
```